src/src/AdminBundle/Controller/EventController.php

```diff
@@ -239,12 +239,7 @@ public function autocompleteOrganizersAction(Request $request)
     {
         $em = $this->getDoctrine()->getManager();
 
-        $organizers = $em->getRepository(Organizer::class)
-            ->createQueryBuilder('o')
-            ->where('o.email LIKE :email')
-            ->setParameter('email', '%'.$request->get('term').'%')
-            ->getQuery()
-            ->getResult();
+        $organizers = $em->getRepository(Organizer::class)->getOrganizersFromAutocomplete($request->get('term'));
```

Comment:
`(Organizer::class)->getOrganizersFrom` nemohlo by sa to volat iba `getAutocomplete...` alebo nejako tak?